config/initializers/resque.rb

```diff
@@ -0,0 +1,2 @@
+Resque.redis = Redis.new(url: ENV['REDIS_URL'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.